controller/src/beerocks/master/wfa_ca.h

```diff
@@ -40,8 +42,15 @@ class wfa_ca {
     static void reply_invalid_error(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
                                     eWfaCaStatus status, const std::string &description);
 
+    static bool reply_running(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx);
+    static void reply_complete(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
+                               const std::string &description = std::string());
+
     static bool reply(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
                       const std::string &reply_string);
+
+    static bool parse_params(std::unordered_map<std::string, std::string> &params,
```

Comment:
I haven't mentioned this in earlier reviews, but we agreed that all new functions and classes should have a doxygen comment.